src/megaapi_impl.cpp

```diff
@@ -23107,6 +23148,128 @@ void MegaApiImpl::sendPendingRequests()
                                                            request->getNodeHandle()));
             break;
         }
+        case MegaRequest::TYPE_BACKUP_FOLDER:
+        {
+            // validate the request
+            if (!request->getFile() || !request->getName()) { e = API_EACCESS; break; }
+
+            // get current user
+            User* u = client->ownuser();
+
+            // get handle of remote "My Backups" folder, from user attributes
+            if (!u || !u->isattrvalid(ATTR_MY_BACKUPS_FOLDER)) { e = API_EACCESS; break; }
+            const string* handleContainerStr = u->getattr(ATTR_MY_BACKUPS_FOLDER);
+            if (!handleContainerStr) { e = API_EACCESS; break; }
+
+            std::unique_ptr<TLVstore> tlvRecords(TLVstore::containerToTLVrecords(handleContainerStr, &client->key));
+            if (!tlvRecords || !tlvRecords->find("h")) { e = API_ENOENT; break; }
+
+            const string& handleStr = tlvRecords->get("h");
+            handle h = MegaApi::base64ToHandle(handleStr.c_str());
+            if (h == UNDEF) { e = API_ENOENT; break; }
+
+            // get Node of remote "My Backups" folder
+            MegaNode* myBackupsNode = getNodeByHandle(h);
+            if (!myBackupsNode) { e = API_ENOENT; break; }
+
+            // get `DEVICE_NAME`, from user attributes
+            if (!u->isattrvalid(ATTR_DEVICE_NAMES)) { e = API_EINCOMPLETE; break; }
+            const string* deviceNameContainerStr = u->getattr(ATTR_DEVICE_NAMES);
+            if (!deviceNameContainerStr) { e = API_EINCOMPLETE; break; }
+
+            tlvRecords.reset(TLVstore::containerToTLVrecords(deviceNameContainerStr, &client->key));
+            if (!tlvRecords || !tlvRecords->find("n")) { e = API_EINCOMPLETE; break; } // is 'n' the right key ??
```

Comment:
No, the right key is the `device-id`. See the corresponding PR: https://github.com/meganz/sdk/pull/2078/files#diff-5d048e8562b9eef0c31b18c647b312f4abf03481607422ffb7f1b12814524fc0R6669